rust/extractor/src/config.rs

```diff
@@ -168,3 +219,10 @@ fn to_cfg_overrides(specs: &Vec<String>) -> CfgOverrides {
         ..Default::default()
     }
 }
+
+fn join_path_buf(lhs: &AbsPath, rhs: &Path) -> AbsPathBuf {
+    let Ok(path) = Utf8PathBuf::from_path_buf(rhs.into()) else {
+        panic!("non utf8 input: {}", rhs.display())
```

Comment:
Question: does `display` work correctly if `rhs` contains non-utf8 data? We wouldn't like to cause an additional  `panic` while trying to display an informative error message.

Reply:
according to https://doc.rust-lang.org/std/path/struct.Path.html#method.display yes, this is safe, and may involve lossy conversion.